service/src/test/java/uk/gov/hmcts/reform/fpl/validators/HasContactDirectionValidatorTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.model.common.Telephone;
+
+import java.util.List;
+
+import java.util.stream.Collectors;
+
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+public class HasContactDirectionValidatorTest {
+    private Validator validator;
+
+    private static final String ERROR_MESSAGE = "Enter the contact's full name";
+
+    @BeforeEach
+    private void setup() {
+        validator = Validation.buildDefaultValidatorFactory().getValidator();
+    }
+
+    @Test
+    void shouldReturnAnErrorIfApplicantsDoesNotExistOnCaseData() {
```

Comment:
Bad name. Validator has nothing to do with case data. It focuses on one field only. Also remove word applicant from case names.

You could change it to: `shouldReturnAnErrorIfTelephoneIsNotPopulated`